napi-inl.h

```diff
@@ -235,14 +240,19 @@ struct FinalizeData {
     });
   }
 
-#ifdef NODE_API_EXPERIMENTAL_HAS_POST_FINALIZER
+#if defined(NODE_API_EXPERIMENTAL_HAS_POST_FINALIZER)
```

Comment:
Same here.

Reply:
Addressed in https://github.com/nodejs/node-addon-api/pull/1568/commits/23b57cc93946df51f30cbfada61213be14f61421